language/move-prover/bytecode-to-boogie/src/prelude.bpl

```diff
@@ -0,0 +1,543 @@
+// ================================================================================
+// Domains
+
+type TypeName;
+type FieldName = int;
+type LocalName;
+type Address = int;
```

Comment:
What is the advantage of defining`Address` as an integer but leaving (e.g.) `LocalName` uninterpreted?

Reply:
There are inconsistencies.  I think it is fine to treat LocalName as an alias for int.